src/Mod/Mesh/Gui/Command.cpp

```diff
@@ -894,9 +894,9 @@ CmdMeshTrimByPlane::CmdMeshTrimByPlane()
 {
     sAppModule = "Mesh";
     sGroup = QT_TR_NOOP("Mesh");
-    sMenuText = QT_TR_NOOP("Trim mesh with a plane");
+    sMenuText = QT_TR_NOOP("Trim With Plane");
     sToolTipText = QT_TR_NOOP("Trims a mesh with a plane");

```

Comment:
```suggestion
    sToolTipText = QT_TR_NOOP("Trims a mesh by removing faces on one side of a selected plane");
```
Extended for clarity. From [docs](https://wiki.freecad.org/Mesh_TrimByPlane) "The Mesh TrimByPlane command trims faces and parts of faces on one side of a plane from a mesh object. "